AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -20,52 +20,22 @@ msgstr ""
 "X-Source-Language: pt_BR\n"
 "X-Generator: Poedit 1.8.7.1\n"
 
-#: AlertaDengue/common/templates/dbf_upload.html:66
-#: AlertaDengue/dbf/templates/dbf_upload.html:66
-msgid "Envie aqui os dados atualizados do SINAN"
-msgstr "Send here the updated SINAN data"
-
-#: AlertaDengue/common/templates/dbf_upload.html:172
-#: AlertaDengue/dbf/templates/dbf_upload.html:172
-msgid "Enviar"
-msgstr "Send"
-
-#: AlertaDengue/common/templates/dbf_upload.html:179
-#: AlertaDengue/common/templates/upload_successful.html:10
-#: AlertaDengue/dbf/templates/dbf_upload.html:179
-#: AlertaDengue/dbf/templates/upload_successful.html:10
-msgid "Ultimos arquivos enviados:"
-msgstr "Last uploaded files:"
-
-#: AlertaDengue/common/templates/upload_successful.html:5
-#: AlertaDengue/dbf/templates/upload_successful.html:5
-msgid "Dados do SINAN recebidos"
-msgstr "SINAN data received"
-
 #: AlertaDengue/dados/charts/cities.py:74
-#, fuzzy
-#| msgid "Limiar epidêmico"
 msgid "limiar pré epidêmico"
 msgstr "Epidemic threshold"
```

Comment:
should be "pre-epidemic threshold"

Reply:
The template [reports by cities] does not seem to be translated yet on the site;(https://github.com/AlertaDengue/AlertaDengue/blob/main/AlertaDengue/dados/templates/report_city.html#L95)

Just the [lines from the charts] module that @ibianka has fuzzy removed.
https://github.com/AlertaDengue/AlertaDengue/blob/main/AlertaDengue/locale/en/LC_MESSAGES/django.po#L45